examples/ljspeech/ljspeech.ipynb

```diff
@@ -0,0 +1,281 @@
+   "source": [
+    "# Download and untar"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "download_and_untar(root_dir)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Prepare audio and supervision manifests"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ljspeech_manifests = prepare_ljspeech(corpus_dir, output_dir)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Extract features"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "feature_set_builder = FeatureSetBuilder(\n",
+    "    feature_extractor=ljspeech_feature_extractor(),\n",
+    "    output_dir=f'{output_dir}/feats'\n",
+    ")\n",
+    "feature_set = feature_set_builder.process_and_store_recordings(\n",
+    "    recordings=ljspeech_manifests['audio'],\n",
+    "    num_jobs=os.cpu_count()\n",
+    ")\n",
+    "ljspeech_manifests['feats'] = feature_set\n",
+    "\n",
+    "cut_set = CutSet.from_manifests(supervision_set=ljspeech_manifests['supervisions'],\n",
+    "                                feature_set=ljspeech_manifests['feats'],\n",
+    "                                recording_set=ljspeech_manifests['audio'])\n",
+    "cut_set.to_json(output_dir / f'cuts.json.gz')\n",
+    "ljspeech_manifests['cuts'] = cut_set"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Make pytorch Dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cuts_train = SpeechSynthesisDataset(ljspeech_manifests['cuts'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Token set"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[' ', '!', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z']\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(cuts_train.targets)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Illustation of an example"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sample = cuts_train[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Tokens"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['I', 'N', ' ', '1', '8', '1', '3']\n"
```

Comment:
I assume this text is typically normalized before training (digits to words)?

Reply:
Text normalization was necessary for conventional TTS pipelines, but some end-to-end TTS systems do not need TN anymore. Anyway, it would be good to allow `lhotse` uses an external TN module.
I plan to allow the function `text_to_tokens()` be overwritten and as a optional parameter of `SpeechSynthesisDataset.__init__()`.